src/cascadia/TerminalApp/ActionAndArgs.cpp

```diff
@@ -100,7 +100,7 @@ namespace winrt::TerminalApp::implementation
         { ToggleCommandPaletteKey, ShortcutAction::ToggleCommandPalette },
         { ToggleFocusModeKey, ShortcutAction::ToggleFocusMode },
         { ToggleFullscreenKey, ShortcutAction::ToggleFullscreen },
-        { TogglePaneZoomKey, ShortcutAction::TogglePaneZoom },
+        // { TogglePaneZoomKey, ShortcutAction::TogglePaneZoom }, // TODO GH#7252: Re-enable pane zooming
```

Comment:
i believe the most minimal change _only_ requires us to comment out this line. Can you validate?

I'd prefer a +1/-1 line change to this file rather than a +4/-4.